src/admin/kubernetes.md

```diff
@@ -0,0 +1,162 @@
+# How to use pgvecto.rs in Kubernetes?
+
+If you want to use `pgvecto.rs` in PostgreSQL which is running in Kubernetes, we provide a tutorial to help you. In this tutorial, we will use [CloudNative-PG](https://cloudnative-pg.io/) to deploy PostgreSQL cluster in Kubernetes.
+
+## Requirements
+
+| Name | Version |
+|------|---------|
+| <a name="requirement_helm"></a> [helm](#requirement\_helm) | >= 2.4.1 |
+| <a name="requirement_kubectl"></a> [kubectl](#requirement\_kubectl) | >= 1.14 |
+| <a name="requirement_kubernetes"></a> [kubernetes](#requirement\_kubernetes) | >= 2.10 |
```

Comment:
There is no kubernetes version `2.10`. The earliest kubernetes version supported by cloudnative-pg is `1.24.0`.

Reply:
Thanks a lot, I fixed it.